src/frontend/sider/Fagsak/Saksoversikt/Behandlinger.tsx

```diff
@@ -31,34 +31,91 @@ const StyledResultatKolonne = styled(Table.HeaderCell)`
     width: 22%;
 `;
 
-interface IBehandlingshistorikkProps {
-    minimalFagsak: IMinimalFagsak;
-}
+const TableHeader = () => {
+    return (
+        <Table.Header>
+            <Table.Row>
+                <StyledOpprettetKolonne children={'Opprettet'} />
+                <Table.HeaderCell children={'Årsak'} />
+                <Table.HeaderCell children={'Type'} />
+                <Table.HeaderCell children={'Behandlingstema'} />
+                <Table.HeaderCell children={'Status'} />
+                <Table.HeaderCell children={'Vedtaksdato'} />
+                <StyledResultatKolonne children={'Resultat'} />
+            </Table.Row>
+        </Table.Header>
+    );
+};
+
+type Props = {
+    fagsakId: number;
+};
+
+export function Behandlinger({ fagsakId }: Props) {
+    const [visHenlagteBehandlinger, setVisHenlagteBehandlinger] = useState(false);
+
+    const {
+        data: minimalFagsak,
+        isPending: hentMinimalFagsakLaster,
+        error: hentMinimalFagsakError,
+    } = useHentFagsak(fagsakId);
```

Comment:
Her burde man nok bare hente kontantsøttebehandlingene ved å innføre en egen react-query hook for det

ba-sak har barnetrygdbehandlinger (ofte bare kalt behandligner i ba-sak)
ks-sak har kontantstøttebehandlinger (ofte bare kalt behandlinger i ks-sak)

```suggestion
    const {
        data: kontantstøttebehandlinger,
        isPending: hentKontantstøttebehandlingerLaster,
        error: hentKontantstøttebehandlingerError,
    } = useHentKontantstøttebehandlinger(fagsakId);
```

I ba-sak la jeg til et endepunkt for å hente barnetrygdbehandlinger, se PRen [her](https://github.com/navikt/familie-ba-sak/pull/5223). Mulig man må gjøre det samme for ks-sak for å hente kontantstøttebehandlinger

Reply:
Jeg tenker det kan tas som en egen oppgave/PR?